webapp/client/src/pages/PersonAHasDisabilityPage.test.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import PersonAHasDisabilityPage from "./PersonAHasDisabilityPage";
+import { Default as StepFormDefault } from "../stories/StepForm.stories";
+
+describe("PersonAHasDisabilityPage", () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      stepHeader: {
+        title: "title",
+      },
+      form: {
+        ...StepFormDefault.args,
+      },
+      fields: {
+        personA_hasDisability: {
+          value: "yes",
+          errors: [],
+        },
+      },
+      prevUrl: "prevUrl",
+      numOfUsers: 1,
+    };
+  });
+
+  it("should render selected value yes", () => {
+    render(<PersonAHasDisabilityPage {...props} />);
+
+    expect(screen.queryAllByRole("radio")[0].checked).toBe(true);
+    expect(screen.queryAllByRole("radio")[1].checked).toBe(false);
+  });
+
+  it("should render selected value no", () => {
+    props.fields.personA_hasDisability.value = "no";
+
+    render(<PersonAHasDisabilityPage {...props} />);
+
+    expect(screen.queryAllByRole("radio")[0].checked).toBe(false);
+    expect(screen.queryAllByRole("radio")[1].checked).toBe(true);
```

Comment:
Could you maybe add different `describes` for the different props? (like `describe("Whith yes preselected" ...` and so on? This way, you'll only have execute `render(<PersonAHasDisabilityPage {...props} />);` once in the befroeEach.